server/server.js

```diff
@@ -1741,7 +1741,7 @@ async function startMonitors() {
     }
 
     for (let monitor of list) {
-        monitor.start(io);
+        await monitor.start(io);
```

Comment:
@chakflying Adding `async` to the function is ok, but I am not sure if adding `await` here or other places are ok or not.

For example, one of monitors gets sucked here and the whole application gets sucked by this monitor. Not quite sure.

Reply:
I see your point.
It's the same behavior as before, as it was with synchronous execution which also was blocked by the monitor start. So I see currently no point in changing this now.

In the future, the startup process could be changed to start all monitors async in the background without waiting. But I think this is out of scope for now in this PR.